### PR TITLE
Add support for binding to svg image

### LINKED ISSUE #14723: Add support to amp-bind for images in svg
At the moment it's not possible to use `amp-bind` to change dynamically the image in a svg.

for example:
```

<!doctype html>
<html ⚡>
<head>
  <meta charset="utf-8">
  <title>My AMP Page</title>
  <link rel="canonical" href="self.html" />
  <meta name="viewport" content="width=device-width,minimum-scale=1">
  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
  <script async src="https://cdn.ampproject.org/v0.js"></script>
  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
  <style amp-custom>
    h1 {
      margin: 1rem;
    }
  </style>
</head>
<body>
  <svg height="100" width="100">
    <image xlink:href="/img/amp-100.jpg" [xlink:href]="/img/amp-200.jpg"></image>
  </svg>
</body>
</html>
```

@choumx FYI

### PR DESCRIPTION
**Pull request by kul3r4**
Fixes  #14723
@choumx PTAL

### CODE REVIEW

File: extensions/amp-bind/0.1/bind-validator.js [hunk 1]
```diff
@@ -302,6 +302,9 @@ function createElementRules_() {
     'FIELDSET': {
       'disabled': null,
     },
+    'image': {
+      'xlink:href': null,
```
Comment (choumx) [inline extensions/amp-bind/0.1/bind-validator.js]: There are security issues with `xlink:href`: https://html5sec.org/#87 

This should be:

```js
'xlink:href': {
  'allowedProtocols': {
    'http': true,
    'https': true,
  },
}
```

And we should add a unit test to bind-validator to make sure the `<image xlink:href="javascript:alert(1)">` will be disallowed.

**Comment by kul3r4** [comment]
@choumx just added the test but it fails, is there anything else I need to change?

**Comment by choumx** [comment]
What's the test failure message? I only see lint errors in the Travis build.

**Comment by kul3r4** [comment]
@choumx Just fixed the lint errors, sorry for the delay

**Comment by kul3r4** [comment]
@choumx do you have any idea why the CI is failing? I can see some errors that don't seem related

**Comment by choumx** [comment]
Might be old failures that have since been fixed. Try `git merge master`.

**Pull request merged**